onnxruntime/python/tools/transformers/fusion_attention.py

```diff
@@ -900,13 +901,17 @@ def create_attention_node(
                 past_kv = self.concat_kv(past_k, past_v)
                 attention_inputs.append(past_kv)
 
-            if add_qk_str is not None:
+            if add_qk_str is not None and has_relative_pos_bias is None:
```

Comment:
has_relative_pos_bias is boolean. Why compare it with None?

Reply:
Yeah will update it